.evergreen/run-ocsp-test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       JDK                     Set the version of java to be used.  Java versions can be set from the java toolchain /opt/java
+#                               "jdk5", "jdk6", "jdk7", "jdk8", "jdk9", "jdk11"
+
+JDK=${JDK:-jdk11}
+
+############################################
+#            Functions                     #
+############################################
+
+provision_ssl () {
+  echo "SSL !"
+
+  # We generate the keystore and truststore on every run with the certs in the drivers-tools repo
+  if [ ! -f client.pkc ]; then
+    openssl pkcs12 -CAfile ${CA_FILE} -export -in ${DRIVERS_TOOLS}/.evergreen/x509gen/client.pem -out client.pkc -password pass:bithere
+  fi
+
+  cp ${JAVA_HOME}/lib/security/cacerts mongo-truststore
+  ${JAVA_HOME}/bin/keytool -importcert -trustcacerts -file ${CA_FILE} -keystore mongo-truststore -storepass changeit -storetype JKS -noprompt
+
+  # We add extra gradle arguments for SSL
+  export GRADLE_EXTRA_VARS="-Pjavax.net.ssl.enabled=true -Pjavax.net.ssl.keyStoreType=pkcs12 -Pjavax.net.ssl.keyStore=`pwd`/client.pkc -Pjavax.net.ssl.keyStorePassword=bithere -Pjavax.net.ssl.trustStoreType=jks -Pjavax.net.ssl.trustStore=`pwd`/mongo-truststore -Pjavax.net.ssl.trustStorePassword=changeit"
```

Comment:
Project properties need extra handling in gradle: See: https://github.com/jstewart-mongo/mongo-java-driver/blob/master/build.gradle#L199-L214 

Ensure all project variables are named correctly and passed through. You can use the short names as per: (https://github.com/jstewart-mongo/mongo-java-driver/blob/master/.evergreen/run-tests.sh#L50)

Reply:
I had originally used short names and decided to go to long names to avoid the extra processing. Since that did not work, I reverted this file to use the short names.